docs/source/developer_resources/Developing_Apps.rst

```diff
@@ -0,0 +1,45 @@
+
+Supported Application Types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Puthon
```

Comment:
Not sure what Puthon is...perhaps better as Python?

Reply:
haha...very funny....DONE